vault/src/faucet.rs

```diff
@@ -21,7 +21,8 @@ where
 
 #[derive(Encode, Decode, Debug, Clone, serde::Serialize)]
 struct FundAccountJsonRpcRequest {
-    pub vault_id: VaultId,
+    pub vault_account_id: AccountId,
+    pub collateral_currency: CurrencyId,
```

Comment:
I noticed this struct didn't match the definition inside the faucet. It would be nice if we could actually test making the http request in an integration test..